main.go

```diff
@@ -43,6 +48,10 @@ var (
 		},
 	)
 	invalidChars = regexp.MustCompile("[^a-zA-Z0-9_]")
+
+	// Udp
+	bindAddress = flag.String("udp.bind-address", ":9122", "Address on which to listen for udp packets.")
+	listenOnUdp = flag.Bool("udp.listen", false, "Listen on udp socket.")
```

Comment:
You can remove this now. I don't really see it happening where having UDP is a problem.